devenv/config.ini

```diff
@@ -1,12 +1,19 @@
 [venv.venv]
-# TODO: need 3.11.8
-python = 3.12.3
+python = 3.11.11
 path = .venv
 # TODO: need to combine requirements files into one
 requirements = requirements.txt
 editable =
   .
 
+[python3.11.11]
+darwin_x86_64 = https://github.com/astral-sh/python-build-standalone/releases/download/20250212/cpython-3.11.11+20250212-x86_64-apple-darwin-install_only.tar.gz
+darwin_x86_64_sha256 = 1f1afc064b523b67c06e6d4b024a8dbb8df5fa12f0f0018b218c855491833451
+darwin_arm64 = https://github.com/astral-sh/python-build-standalone/releases/download/20250212/cpython-3.11.11+20250212-aarch64-apple-darwin-install_only.tar.gz
+darwin_arm64_sha256 = 6dd5603e5570b8c4e1abf1ef2a329f195dd052890b1fa3b01b7406552a4f45a1
+
+
+
 [python3.12.3]
```

Comment:
as this is now unused you can remove it. will probably be outdated when you do upgrade to 3.12.x